e2e/actions/l2_challenger_test.go

```diff
@@ -148,12 +172,15 @@ func TestChallenger(gt *testing.T) {
 	require.Equal(t, types.ReceiptStatusSuccessful, receipt.Status, "failed to create challenge")
 
 	// check challenge created
-	colosseumContract, err := bindings.NewColosseum(sd.DeploymentsL1.ColosseumProxy, miner.EthClient())
-	require.NoError(t, err)
 	challenge, err := colosseumContract.GetChallenge(nil, outputIndex)
 	require.NoError(t, err)
 	require.NotNil(t, challenge, "challenge not found")
 
+	// check bond amount doubled
```

Comment:
(Same in `TestChallengerChallengerTimeout` function.)
```suggestion
	// check bond amount doubled after create challenge
```